src/components/earn/StakedAmountsHelper.tsx

```diff
@@ -0,0 +1,20 @@
+import { TokenAmount } from '@ubeswap/sdk'
+import QuestionHelper from 'components/QuestionHelper'
+import React from 'react'
+
+interface Props {
+  userAmountTokenA: TokenAmount | undefined
+  userAmountTokenB: TokenAmount | undefined
+}
+
+export default React.memo(function StakedAmountsHelper({ userAmountTokenA, userAmountTokenB }: Props) {
```

Comment:
Is there a particular reason to use React.memo here?

Reply:
not in this case